examples/mayavi/mlab/protein.py

```diff
@@ -73,17 +73,17 @@
 last_chain_label = None
 for line in infile:
     line = line.split()
```

Comment:
This old example fails for me to, but I'd rather suggest converting the line in one go and drop the other changes. E.g.

line = line.decode("utf-8")

Reply:
Yes I agree, this is much easier and works.